Gemfile.lock

```diff
@@ -162,22 +165,21 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  aws-sdk-ec2 (~> 1)
+  aws-sdk-ec2
   byebug
   climate_control
   foreman
-  govuk_app_config (~> 2.2)
-  govuk_message_queue_consumer (~> 3.5)
-  govuk_test (~> 1.0)
-  plek (~> 4.0)
-  pry (~> 0.13)
-  rake (~> 13.0)
-  rspec-core (~> 3.9)
-  rspec-expectations (~> 3.9)
-  rspec-mocks (~> 3.9)
+  govuk_app_config
+  govuk_message_queue_consumer
+  govuk_test
+  plek
+  pry
+  rake
+  rspec-core
+  rspec-expectations
+  rspec-mocks
   rubocop-govuk
+  thwait
   timecop
   webmock
 
-BUNDLED WITH
-   1.17.2
```

Comment:
Did you mean to get rid of this altogether?

Reply:
I've reinstated it and updated to 1.17.3